### PR TITLE
Blackfiction

### PR DESCRIPTION
**Pull request by Blackfiction**
updated nl strings

### CODE REVIEW

**Comment by mherger** [comment]
Way to go! This one looked good 😀

**Comment by Blackfiction** [comment]
Mmh, I closed it because I saw 4 topics where I think I only want to commit 1 topic.
I only would like to add this one:
***@***.***<https://github.com/Blackfiction>
updated nl strings<https://github.com/Logitech/slimserver/pull/741/commits/c657468babee6cd012e45b4cca79709504a23485>
c657468<https://github.com/Logitech/slimserver/pull/741/commits/c657468babee6cd012e45b4cca79709504a23485>

Should I reopen the pull request?
Sorry that I am not that familiar with github…

Daan.

**Comment by michaelherger** [comment]
Before you submit the PR, re-base your branch on top of the latest 8.3. I think you merged 8.3 to it instead. Both ways should result in the same code, but re-basing results in the cleaner history you want.

**Comment by Blackfiction** [comment]
I opened it again as the rebase mentioned that all was up to date. Hope all is ok now?
Daan.

File: strings.txt [hunk 1]
```diff
@@ -8429,24 +8436,28 @@ SETUP_ENABLE_STATISTICS
...
 SETUP_ENABLE_PERSISTENTHTTP
 	EN	Persistent mode
 	FR	Mode persistent
+	NL	Persistent-modus
 
 SETUP_ENABLE_BUFFEREDHTTP
 	DE	HTTP(S) Datenströme zwischenspeichern
 	EN	Cache HTTP(S) streams on disk
-	FR	Cacher les flux HTTP(S) sur disque
+	FR	Cache-modus
```
Comment (michaelherger) [inline strings.txt:8455]: Please do not modify languages other than NL.

File: strings.txt [hunk 2]
```diff
@@ -13759,8 +13780,8 @@ PLAYLIST_INFO
 	ES	Información de lista de reproducción
 	FI	Soittoluettelotiedot
 	FR	Infos sur la liste de lecture
-	IT	Info playlist
-	NL	Playlist-info
+	IT	Playlist Info
```
Comment (michaelherger) [inline strings.txt:13783]: Please do not modify languages other than NL.

File: strings.txt [hunk 3]
```diff
@@ -15780,7 +15802,7 @@ SEARCHFOR_ARTISTS
 	HE	איתור מבצעים
 	IT	Cerca artisti
 	JA	アーチストを検索する
-	NL	Artiesten zoeken
+	c
```
Comment (michaelherger) [inline strings.txt]: Accidental change?

File: strings.txt [hunk 4]
```diff
@@ -15792,6 +15814,7 @@ SEARCHFOR_ARTISTS_DBL
 	ES	Buscar artistas
 	FR	Cherche artistes
 	PL	Wyszukaj wykonawców
+	NL	Artiesten zoeken
```
Comment (michaelherger) [inline strings.txt:15817]: Please keep the order of the languages alphabetically. Thanks!

File: strings.txt [hunk 5]
```diff
@@ -15859,6 +15883,7 @@ SEARCHFOR_SONGS_DBL
 	ES	Buscar Canciones
 	FR	Cherche morceaux
 	PL	Wyszukaj utwory
+	NL	Nummers zoeken
```
Comment (michaelherger) [inline strings.txt:15886]: Language order.

File: strings.txt [hunk 6]
```diff
@@ -15835,6 +15858,7 @@ SEARCHFOR_ALBUMS_DBL
 	ES	Buscar álbumes
 	FR	Cherche albums
 	PL	Wyszukaj albumy
+	NL	Albums zoeken
```
Comment (michaelherger) [inline strings.txt:15861]: Language order.

**Review by michaelherger — CHANGES_REQUESTED** [review]
I've found some minor issues in the main strings.txt. Please try not to change anything unnecessary, other languages, string order etc. Thanks a lot!

**Comment by Blackfiction** [comment]
Ok! Did not know about the language order and the c was an accident.

I have added strings.txt for MAI as well as I can not find it in github/lms plugins. Could you please merge that one as well?
Thanks, Daan.

File: strings.txt (hunk 3 above)
Comment (Blackfiction) [reply strings.txt]: yes, sorry

**Comment by Blackfiction** [comment]
Do I need to change these or can you do that?
Thx Daan

**Comment by michaelherger** [comment]
You can change whatever is needed, commit again. Then push to the same branch again. This should update the PR.

**Comment by Blackfiction** [comment]
Ok, I think I fixed your comments, Daan.

File: strings.txt [hunk 7]
```diff
@@ -9515,7 +9532,7 @@ SETUP_SN_REPORT_STATS_DESC
...
 	IT	Al fine di migliorare mysqueezebox.com, è possibile inviare dati statistici sull'ascolto di stazioni radio in Internet e sui servizi musicali utilizzati. Il trattamento di tali dai, raccolti collettivamente e in modo anonimo, è regolato dall'<a href="http://www.logitech.com/index.cfm/footer/privacy/&cl=it,it" target="_blank">informativa sulla privacy</a>.
-	NL	Help met de verbetering van mysqueezebox.com door statistieken over het luisteren naar internetradio en muziekservices te rapporteren. (Deze anonieme informatie wordt globaal verzameld en wordt beschermd door ons <a href="http://www.logitech.com/index.cfm/footer/privacy/&cl=us,en" target="_blank">privacybeleid</a>.)
+	NL	Help met de verbetering van mysqueezebox.com door statistieken over het luisteren naar internetradio en muziekservices te rapporteren. Deze anonieme informatie wordt globaal verzameld en wordt beschermd door ons <a href="http://www.logitech.com/index.cfm/footer/privacy/&cl=us,en" target="_blank">privacybeleid</a>.)
```
Comment (mherger) [inline strings.txt:9535]: Accidental change? You remove the opening bracket, but not the closing one?

File: strings.txt [hunk 8]
```diff
@@ -9530,7 +9547,7 @@ SETUP_SN_REPORT_STATS_ENABLE
 	FI	Kyllä, auta parantamaan mysqueezebox.com-verkkoa raportoimalla tilastot.
 	FR	Autoriser la collecte d'informations pour contribuer à l'amélioration de mysqueezebox.com.
 	IT	Sì, autorizzo la raccolta dei dati a fini statistici per migliorare mysqueezebox.com.
-	NL	Ja, statistieken rapporteren om mysqueezebox.com te verbeteren.
+	NL	Ja, statistieken rapporteren.
```
Comment (mherger) [inline strings.txt:9550]: Please leave the information about mysqueezebox.com in. The user needs to know what the data would be collected for.

**Review by mherger — CHANGES_REQUESTED** [review]
Some more tweaks...

**Comment by Blackfiction** [comment]
Thanks, fixed and submitted!

**Comment by mherger** [comment]
Thanks a lot!

**Pull request merged**